mmpose/datasets/transforms/formatting.py

```diff
@@ -182,12 +148,11 @@ def transform(self, results: dict) -> dict:
 
         # pack instance data
         gt_instances = InstanceData()
+        instance_mapping_table = results.get('instance_mapping_table', None)
+        if instance_mapping_table is not None:
+            self.instance_mapping_table = instance_mapping_table
```

Comment:
I suggest not directrly modifying a class attribute, especially replacing it. How about create a new temporal mapping table that initialized from `self.instance_mapping_table`, and then update it via `result['instance_mapping_table']`